bigchaindb/commands/bigchaindb.py

```diff
@@ -191,6 +191,29 @@ def run_upsert_validator_approve(args, bigchain):
         return False
 
 
+def run_upsert_validator_show(args, bigchain):
+    """Retrieves information about an upsert-validator election
+
+    :param args: dict
+        args = {
+        'election_id': the transaction_id for an election (str)
+        }
+    :param bigchain: an instance of BigchainDB
+    :return: (public_key, power, node_id) for the proposed change to the validator set
+    """
+
+    election = bigchain.get_transaction(args.election_id)
+
+    new_validator = election.asset['data']
+
+    public_key = new_validator['public_key']
+    power = new_validator['power']
+    node_id = new_validator['node_id']
+    status = election.get_status(bigchain)
+
+    return f'public_key={public_key}\npower={power}\nnode_id={node_id}\nstatus={status}'
```

Comment:
I don't think returning the value will print it on the console?

Reply:
Done